lib/mariaex.ex

```diff
@@ -188,10 +190,10 @@ defmodule Mariaex do
       :text ->
         query = %Query{type: :text, name: name, statement: statement,
                        ref: make_ref(), num_params: 0}
-        {:ok, query}
+        {:ok, Mariaex.Protocol.sanitize_query(query, conn)}
       type when type in [:binary, nil] ->
         query = %Query{type: type, name: name, statement: statement}
-        DBConnection.prepare(conn, query, opts)
+        DBConnection.prepare(conn, Mariaex.Protocol.sanitize_query(query, conn), opts)
```

Comment:
All sanitization could occur in the protocol module, we shouldnt need either of these calls.

Reply:
Resolved.